src/main/java/racingcar/domain/Car.java

```diff
@@ -20,21 +19,22 @@ public void oneTurn(int randomTurnNumber) {
     }
 
     private void progressPosition() {
-        this.position++;
+        Position carPosition = this.position;
+        carPosition.progressOneTurn();
     }
 
     public boolean isSamePosition(int targetPosition) {
-        if (this.position == targetPosition) {
-            return true;
-        }
-        return false;
+        Position carPosition = this.position;
+        return carPosition.isSameValue(targetPosition);
```

Comment:
아래 처럼 호출하지 않는 이유가 있을까요? 
```suggestion
        return this.position.isSameValue(targetPosition);
```

Reply:
처음에는 this.position으로 바로 리턴하게 되었으나, .이 너무 많이찍히는 것 같아서, 바꾸게 되었습니다.
하지만 말씀해주신데로, .을 추가하여 진행하는 것이 조금 더 맞을 것 같습니다!